ppl/lake/ztests/import-check.yaml

```diff
@@ -1,14 +1,14 @@
 script: |
   mkdir logs
-  zar import -R logs missingcol.tzng
+  zar import -R logs missingcol.zson
 
 inputs:
-  - name: missingcol.tzng
+  - name: missingcol.zson
     data: |
-      #0:record[a:string,b:string]
-      0:[a;]
+      {a:"a",b:"b"} (=foo)
+      {a:"a"} (foo)
 
 outputs:
   - name: stderr
     regexp: |
-      .*record missing a field
+      .*record decorator columns \(2\) mismatched with value columns \(1\)
```

Comment:
This was the one test where I made a material change to achieve what seemed to be an equivalent test with ZSON input data. Feel free to propose adjustments or advocate that I leave the TZNG one for now.

Reply:
That error message looks like it could be improved but we can put that in another issue...

Reply:
Thanks @mccanne. I opened #2394 regarding the error message.